selfdrive/ui/translations/main_ar.ts

```diff
@@ -0,0 +1,1347 @@
+        <translation>إلى اليمين</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="211"/>
+        <source>Are you sure you want to reboot?</source>
+        <translation>هل أنت متأكد أنك تريد إعادة التشغيل؟</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="218"/>
+        <source>Disengage to Reboot</source>
+        <translation>فك الارتباط لإعادة التشغيل</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="224"/>
+        <source>Are you sure you want to power off?</source>
+        <translation>هل أنت متأكد أنك تريد إيقاف التشغيل؟</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="231"/>
+        <source>Disengage to Power Off</source>
+        <translation>فك الارتباط لإيقاف التشغيل</translation>
+    </message>
+</context>
+<context>
+    <name>DriveStats</name>
+    <message>
+        <location filename="../qt/widgets/drive_stats.cc" line="37"/>
+        <source>Drives</source>
+        <translation>أرقام القيادة</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/drive_stats.cc" line="39"/>
+        <source>Hours</source>
+        <translation>ساعات</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/drive_stats.cc" line="44"/>
+        <source>ALL TIME</source>
+        <translation>في كل وقت</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/drive_stats.cc" line="46"/>
+        <source>PAST WEEK</source>
+        <translation>الأسبوع الماضي</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/drive_stats.h" line="15"/>
+        <source>KM</source>
+        <translation>كم</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/drive_stats.h" line="15"/>
+        <source>Miles</source>
+        <translation>اميال</translation>
+    </message>
+</context>
+<context>
+    <name>DriverViewScene</name>
+    <message>
+        <location filename="../qt/offroad/driverview.cc" line="55"/>
+        <source>camera starting</source>
+        <translation>بدء تشغيل الكاميرا</translation>
+    </message>
+</context>
+<context>
+    <name>InputDialog</name>
+    <message>
+        <location filename="../qt/widgets/input.cc" line="71"/>
+        <source>Cancel</source>
+        <translation>إلغاء</translation>
+    </message>
+    <message numerus="yes">
+        <location filename="../qt/widgets/input.cc" line="168"/>
+        <source>Need at least %n character(s)!</source>
+        <translation type="unfinished">
+            <numerusform></numerusform>
+            <numerusform></numerusform>
+            <numerusform></numerusform>
+            <numerusform></numerusform>
+            <numerusform></numerusform>
+            <numerusform></numerusform>
```

Comment:
Do you mind adding all the plural forms for these translations?

Reply:
Done! I also see 6 vanished translations, should we remove them?

Reply:
Yep that's fine, you can run `./update_translations.py --vanish`